conf/application.conf

```diff
@@ -82,7 +82,7 @@ application.langs="en"
 # you may need to define a router file `conf/my.application.routes`.
 # Default to Routes in the root package (and conf/routes)
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
-application.router=prod.Routes
+play.http.router=prod.Routes
```

Comment:
You can also remove the following in this file:
```
# Root logger:
logger.root = ERROR

# Logger used by the framework:
logger.play = INFO

# Logger provided to your application:
logger.application = DEBUG
```
as this gives a warning when compiling i.e. `Logger configuration in conf files is deprecated and has no effect. Use a logback configuration file instead.`